templates/diskovc/diskovc.py

```diff
@@ -0,0 +1,162 @@
+from js9 import j
+from zerorobot.template.base import TemplateBase
+
+class Diskovc(TemplateBase):
+
+    version = '0.0.1'
+    template_name = "diskovc"
+
+    OVC_TEMPLATE = 'github.com/openvcloud/0-templates/openvcloud/0.0.1'
+    SSH_TEMPLATE = 'github.com/openvcloud/0-templates/sshkey/0.0.1'
+    ACCOUNT_TEMPLATE = 'github.com/openvcloud/0-templates/account/0.0.1'
+
+    def __init__(self, name, guid=None, data=None):
+        super().__init__(name=name, guid=guid, data=data)
+        
+        self.data['devicename'] = name
+        self._ovc = None
+        self._account = None
+    
+    def validate(self):
+        data = self.data
+
+        # Get object for an OVC service, make sure exactly one is running
+        ovcs = self.api.services.find(template_uid=self.OVC_TEMPLATE, name=data.get('openvcloud', None))
+        if len(ovcs) != 1:
+            raise RuntimeError('found %s openvcloud connections, requires exactly 1' % len(ovcs))
+        data['openvcloud'] = ovcs[0].name
```

Comment:
Same remark as before, this is not needed since you use data['openvcloud'] in the find call